hw/ip/sysrst_ctrl/data/sysrst_ctrl.hjson

```diff
@@ -14,7 +14,7 @@
     "rst_slow_ni"
   ],
   interrupt_list: [
-    { name: "rbox_intr_o",
+    { name: "sysrst_ctrl_intr_o",
```

Comment:
I didn't expect the interrrupt name should have `intr_o` suffix. could you please double check?

Reply:
Is there any naming convention? This is a combined interrupt from sysrst_ctrl. The specific cause/status is defined in combo/key_intr_status.

Reply:
I meant, the interrupt name should be without `intr_` prefix and `_o` suffix. Look at other HWIPs and compare `.hjson` and IP top `.sv` to see how the interrupt name is converted into signal port name.